common/src/main/java/dev/architectury/registry/ReloadListenerRegistry.java

```diff
@@ -20,15 +20,28 @@
 package dev.architectury.registry;
 
 import dev.architectury.injectables.annotations.ExpectPlatform;
+import net.minecraft.resources.ResourceLocation;
 import net.minecraft.server.packs.PackType;
 import net.minecraft.server.packs.resources.PreparableReloadListener;
 
+import javax.annotation.Nullable;
```

Comment:
Do we use javax nullable?

Reply:
We do not, please switch to jbr annotations if possible